rhessys/tec/input_new_patch_mult.c

```diff
@@ -42,7 +42,7 @@
 #include "rhessys.h"
 #include "phys_constants.h"
 #define ONE 1.0
-
+#include "params.h"
 
  void  input_new_patch_mult(
```

Comment:
@xiaoli-chen What I am saying is that if possible, the code to read patch variables should be put into its own function, and then this function should be used by both input_new_patch.c and input_new_patch_mult.c, instead of duplicating this code in both places.  That's all.